rgw/v2/tests/s3cmd/test_rgw_ops_log.py

```diff
@@ -113,12 +126,79 @@ def test_exec(config, ssh_con):
             )
             uploaded_file = uploaded_file_info["name"]
             log.info(f"Uploaded file {uploaded_file} to bucket {bucket_name}")
-            opslog_out = subprocess.getoutput(
-                "timeout 1 nc -U --recv-only /var/run/ceph/*/opslog"
+            pr = subprocess.Popen(
+                f"timeout 1 nc -U --recv-only /var/run/ceph/{ceph_detail['fsid']}/opslog",
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+                universal_newlines=False,
+                shell=True,
             )
-            print(f"print opslog for put object {opslog_out}")
-
-    # check for any crashes during the execution
+            opslog_out, _ = pr.communicate()
+            opslog_out = opslog_out.decode("utf-8", errors="ignore")
+            opslog_out = str(opslog_out).split(",\n")[0] + "]"
+            data = json.loads(opslog_out)
+            log.info(f"opslog for operation upload object is {data}")
+            log.info(f"Uri is {data[0]['uri']}")
+
+            # s3:operation - list objects in bucket
+            log.info("list all objects in bucket operation")
+            utils.exec_shell_cmd(f"/home/cephuser/venv/bin/s3cmd ls s3://{bucket_name}")
+
+            pr = subprocess.Popen(
+                f"timeout 1 nc -U --recv-only /var/run/ceph/{ceph_detail['fsid']}/opslog",
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+                universal_newlines=False,
+                shell=True,
+            )
+            opslog_out, _ = pr.communicate()
+            opslog_out = opslog_out.decode("utf-8", errors="ignore")
+            opslog_out = str(opslog_out).split(",\n")[0] + "]"
+            data = json.loads(opslog_out)
+            log.info(f"opslog for list_object is : {data}")
+            log.info(f"Uri is {data[0]['uri']}")
+
+    if config.test_ops.get("rgw_log_detail", False):
+        log.info("Verify rgw logs exist and rgw log details")
+        ceph_version_id, _ = utils.get_ceph_version()
+        rgw_log_dict = {}
+        rgw_log_dict[ceph_version_id] = {}
+        root_path = str(Path.home()) + "/rgw_log_details.yaml"
+        log.info(f"Verify file exist {root_path}, if not create file")
+        if not os.path.exists(root_path):
+            utils.exec_shell_cmd(f"touch {root_path}")
```

Comment:
@anrao19 
as an enhancement in the coming PR's, you could add code in the else part here to compare the previous log size with the current log size of every rgw daemon on every host (if greater than one GB)

we can get only the size of a file using this
```
[root@ceph-pri-hsm-ms-izl3ix-node5 ~]# stat -c %s /var/log/ceph/0c1b712c-391b-11ef-a58e-fa163e71c208/ceph-client.rgw.shared.pri.ceph-pri-hsm-ms-izl3ix-node5.hegihi.log
24743703498
[root@ceph-pri-hsm-ms-izl3ix-node5 ~]#```

Reply:
Sure @hmaheswa